dpdk/vr_dpdk_virtio.c

```diff
@@ -506,6 +548,7 @@ dpdk_virtio_to_vm_flush(void *arg)
                             rte_pktmbuf_data_len(vq->vdv_tx_mbuf[i]);
         }
 
+        DPDK_VIRTIO_WRITER_STATS_PKTS_IN_ADD(vq, 1);
```

Comment:
Would be more efficient to call this once outside the loop (with num_pkts as argument).